Ms2Database/DB Classes/Account.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ms2Database.DB_Classes
+{
+    public class Account
+    {
+        public Account()
```

Comment:
Can the constructor be after the fields? Also, can an empty line be left in between the fields for readability?

Reply:
It can only be above the fields. empty lines have been added in between the fields for readability